### PR TITLE
WIP: 1.x Release Branch

### PR DESCRIPTION
**Pull request by wesleytodd**
The goal of for the 1.x release is parity with `npm init -y` with a few better defaults which were already feature complete. Things that need to be done to achieve this:

- [x] A test which exercises npm init as a comparison
  - [x] From that we should be able to make a better list
- [x] Documentation
- [ ] what else?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,16 +1,18 @@
...
   "description": "Create a package.json",
   "main": "index.js",
   "type": "commonjs",
   "bin": {
-    "create-package-json": "./bin/create-package-json"
+    "create-package-json": "bin/create-package-json"
   },
   "scripts": {
     "test": "standard && mocha",
+    "test:debug": "mocha --inspect --inspect-brk --timeout=0",
```
Comment (boneskull) [inline package.json:12]: `--timeout=0` should be unnecessary, as Mocha should recognize `--inspect`/`--inspect-brk` and set this automatically

File: package.json [hunk 2]
```diff
@@ -31,16 +33,18 @@
   },
   "license": "ICS",
```
Comment (boneskull) [inline package.json:34]: should be `MIT` or `Apache-2.0` unless we have a good reason to use ISC.  based on our conversation, I don't think we do :D

File: package.json (hunk 1 above)
Comment (wesleytodd) [reply package.json:12]: I never new this and have it all over my projects!  Is this new? I know I added this in response to tests timing out in while I was at a break point.

**Comment by wesleytodd** [comment]
@Ethan-Arrowood guessing you are not a member of this org, but would you mind giving this a once over?

This is barely more than `npm init` but I think that is fine for us to cut the 1.x. I have tested with both the programmatic api and the cli in a few projects. And since it is a 1.0.0 I am comfortable just publishing but thought I would get your input first.  Same goes for anyone else who wants to review this, but Ethan and I discussed this as part of the package metadata collab space.

**Comment by Ethan-Arrowood** [comment]
Sure! We could probably Bikeshed a lot of this (typescript, esm, dependencies/dependency less, node version support). But I'm all about iteration so we should merge this and work from there.

**Review by Ethan-Arrowood — APPROVED** [review]

**Comment by wesleytodd** [comment]
Yeah, I would prefer we avoid much bikeshedding if we can. That said, just to address a few things:

Especially for small tools like this, simplicity is key. So:

1. typescript: We wouldn't get much internal value out of types because it should be a tiny module. I do think we should add jsdoc annotations and generate a `.d.ts`.
2. ESM: since the complexity to dual publish is still high and ESM consumers can easily work around a cjs package I see this as a simplicity play as well.
3. dependencies: this for sure can be cleaned up imo. But right now maybe not necessary.
4. node versions: So I went with current (and using the nv tool to keep that updated). Ideally we would keep support to just node lts versions at the time of publish I think

**Pull request merged**